src/omv/sensors/paj6100.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * This file is part of the OpenMV project.
+ * Copyright (c) 2019 Lake Fu <lake_fu@pixart.com>
+ * This work is licensed under the MIT license, see the file LICENSE for details.
+ *
+ * PAJ6100 driver.
+ *
+ */
+#ifndef __PAJ6100_H__
+#define __PAJ6100_H__
+#include <stdbool.h>
```

Comment:
```suggestion
```

Reply:
Yes, I need it. 
Because after 2 lines, I declare the function
`bool paj6100_detect(sensor_t *sensor) { return false; }`
It will need "bool".

Reply:
Hi it's included in sensor.c before this header is included... I try to move to *.c files instead of .h files.

Reply:
Are you sure? Implicit include is not a good idea, some compiler will throw error/warning for this, specially GCC.

Reply:
Yes it's okay, the stdbool is included before all sensor headers in sensor.c